src/components/utils/utils.js

```diff
@@ -59,3 +59,9 @@ export const postComment = (article_id, comment) => {
       throw error;
     });
 };
+
+export const getTopics = () => {
+  return api.get(`/topics`).then((response) => {
+    return response.data.topics;
+  });
+};
```

Comment:
As mentioned earlier, you will need to fetch the articles by topic from the API here.